### ISSUE TITLE
Examples of using Roswell in AppVeyor

### ISSUE THREAD

**Issue by elliottslaughter** [roswell/roswell#344]
Are there any existing examples of successful use of Roswell from inside AppVeyor? I'm sure this is all my fault, but I'm having a very hard time figuring out all the appropriate escaping rules to make things work properly.

Here's an example Roswell script I'm trying to run at the moment:

https://github.com/elliottslaughter/ip-interfaces/blob/7fa5540c66d45fcc8a708edeed123fd8fe26b2b8/appveyor.yml

I can't even get a basic `(print "asdf")` to work. If I print the string with `echo`, it seems to be showing what I expect, but then when I run it through Roswell it doesn't work. Example output:

https://ci.appveyor.com/project/elliottslaughter/ip-interfaces/builds/19811878

```
echo '(print "asdf")'
(print "asdf")
ros run -e '(print "asdf")' -q
Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.
restarts (invokable by number or by possibly-abbreviated name):
  0: [CONTINUE   ] Retry using ASDF.
  1: [USE-VALUE  ] Use specified value.
  2: [STORE-VALUE] Set specified value and use it.
  3:               Ignore runtime option --eval "(ros:run '((:eval\"(ros:quicklisp)\")(:eval \"(print asdf)\")(:quit)))".
  4: [ABORT      ] Skip rest of --eval and --load options.
  5:               Skip to toplevel READ/EVAL/PRINT loop.
  6: [EXIT       ] Exit SBCL (calling #'EXIT, killing the process).
ros : 
At line:1 char:1
+ ros run -e '(print "asdf")' -q
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
debugger invoked on a UNBOUND-VARIABLE in thread
#<THREAD "main thread" RUNNING {10027900C3}>:
  The variable ASDF is unbound.
```

You'll note I'm using PowerShell in my AppVeyor script; I've given up on `cmd.exe` because the escaping rules there seem to be literally insane.

I'll probably figure out my problem eventually, but I could save a lot of time if I had examples to go off of.

Thanks in advance!

**Comment by elliottslaughter** [comment]
Well of course right after posting this issue I figured out what the problem was.

Apparently it's not enough to just escape for the shell (batch or PowerShell), because double quotes are *also* interpreted by the program that you're calling. In this case, it seems that `\"` is sufficient to convince Roswell to take the quotes as quotes. But from the perspective of PowerShell, I'm literally passing a `\` followed by a `"`. Yuck.

Anyway, here's my working AppVeyor script, in case anyone else is interested:

https://github.com/elliottslaughter/ip-interfaces/blob/master/appveyor.yml

TL;DR: Use PowerShell (batch escaping is insane), but also escape double quotes as needed for the underlying application.
